src/api/model/user.model.ts

```diff
@@ -1,9 +1,12 @@
+import { ConsultationModel } from './consultation.model'
+import { RatingModel } from './rating.model'
 export interface UserModel {
   id: number
   authSchId: string
   firstName: string
   lastName: string
   email: string
-  // ownedGroups: ids
+  ownedConsultations?: ConsultationModel[]
   // memberships: ids
+  ratings?: RatingModel[]
```

Comment:
These fields don't have to be optional, if the user doesn't have any consultations or ratings, they're going to be an empty array